pandas/core/frame.py

```diff
@@ -4353,14 +4353,157 @@ def duplicated(self, subset=None, keep='first'):
               first occurrence.
             - ``last`` : Mark duplicates as ``True`` except for the
               last occurrence.
-            - False : Mark all duplicates as ``True``.
+            - False : Mark all duplicates as ``True``. This option is not
+              compatible with ``return_inverse``.
+        return_inverse : boolean, default False
+            If True, also return the selection from the index from the
```

Comment:
"selection from the index" sounds strange to me, and it does not really say what the values are (I understand what you mean, but I think we should think about a better wording). 
Also the following lines are rather complex to understand (eg the "boolean complement of the first output")